src/app/payments/helpers.ts

```diff
@@ -130,6 +130,44 @@ const recipientDetailsFromInvoice = async <R extends WalletCurrency>(
   }
 }
 
+const checkLimitsBase = async <S extends WalletCurrency>({
```

Comment:
if this is to check account volumes then we should pass account id or account. Also, this is the problem with "helpers" now we keep adding new functions and increasing the complexity of a file that give no context

Reply:
Good catch, I changed the arg to `accountId` here: https://github.com/GaloyMoney/galoy/pull/1973/commits/6062aa743c33fec0d2176da8abac9745bce8ed24

Good point on the helpers thing too. I think these can probably be their own use-case module now. I can change that in the next PR #1953 where I'm already adding new use-cases and doing some refactoring